compiler/rustc_codegen_llvm/src/back/owned_target_machine.rs

```diff
@@ -71,7 +71,7 @@ impl OwnedTargetMachine {
                 split_dwarf_file.as_ptr(),
                 output_obj_file.as_ptr(),
                 debug_info_compression.as_ptr(),
-                force_emulated_tls,
```

Comment:
You need to modify the C++ code that handles this: currently `force_emulated_tls = false` means "use the default setting for this target". We want to change this to always force the given setting for emulated TLS.